Gopkg.toml

```diff
@@ -26,12 +26,12 @@
   version = "1.1.12"
 
 [[constraint]]
-  branch = "master"
   name = "github.com/pilosa/go-pilosa"
+  branch = "master"
 
 [[constraint]]
   name = "github.com/pilosa/pilosa"
-  version = "0.3.1"
+  branch = "dep-fixes"
```

Comment:
yes, good call, but we need to merge https://github.com/pilosa/pilosa/pull/887 first - for some reason, Pilosa was constraining boltdb to a revision instead of a version which was causing unresolvable dependency issues with pdk

Reply:
and then I guess we'll need to depend on branch = "master" since that will be the only thing which has the dep fix until we cut a new release